Sources/RoktUXHelper/RoktUX.swift

```diff
@@ -126,9 +130,12 @@ public class RoktUX: UXEventsDelegate {
                     let layoutLoader = defaultLayoutLoader ?? layoutLoaders?
                         .first { $0.key == layoutPlugin.targetElementSelector }?
                         .value
+                    let layoutPluginViewState = try? layoutPluginViewStates?
```

Comment:
whats the try for?

Reply:
oops that must have been before i changed this implementation. removed